### DISCUSSION TITLE
Community Discussion: Road Plots

### DISCUSSION THREAD

**Discussion by Phyther** [TownyAdvanced/Towny#2960] [Ideas]
Would it be possible to add a "Road" plot type that can be claimed with /t claim road and toggleable in the config (so some folks can turn it off) so that mayors can build roads between towns?

**Comment by LlmDl** [comment]
I like this idea, but it needs some serious fleshing-out.

The first thing a special plot type should have is a purpose, something that makes its existence more than just role-playing. In the case of a Road plot it's obviously going to be used between towns. Some of the ways I can see this being special are the following:
- Road plots ignore the min-distance-between-town-claims.
  - Problem: what would make this non-abusable if there's a simple way to claim near to a town where you ought not be able to.
- Road plots could be owned by the nation.
  - Problem: this would be the first occurrance of a nation owning land. Maybe this could be owned by the capital city?
- Maybe road plots are limited in amount like outposts using the townLevels, maybe they're only given to towns in nations?
  - Problem: if they're only given to nations' towns then how do we handle road plots when a town leaves the nation?

I'd like to hear some other ideas from the community on this one before it gets accepted.

**Comment by BeeRando** [comment]
A simple fix would be just let it be an admin plot type. People can make their own roads, and just have a staff member claim the road. Staff could just fly around the world, autoclaiming roads as they fly over either ones they made, or ones players made, and bam. Roads have no set minclaim effect, and can be overclaimed by anyone. This protects the road from greif, as people can't break it, but also don's limit anyone from claiming that area, or an area near it.

**Comment by IdaEdz** [comment]
Permission based system, where set out players can be "road builders and claimers".
These players (possibly hired by a town or nation) would be able to build a nice road and then claim it. Give it a name for the town/nation. Either themself can unclaim or you have seperate class which can remove roads. 
Server admins would always be able to override with higher permissions.

**Comment by Haaruun-I** [comment]
I think there shuld be a town type or permission like "Allow builds nearby"  so you as the mayor can do "/t allow (townname) claimnear" or something. this might be usefull if you also can force member towns of your nation to accept this or something like "/n allow (townname) claimnear all"

**Comment by MilesBHuff** [comment]
We could also make it where nations can buy and use bonus plots (so nations would almost be their own towns, in some sense).  Roads could then just be a plot type that you set with `/plot set road`.  Any town inside a nation could claim these nation-owned plots if they need to expand over them.

**Comment by DragonisOmega** [comment]
I could see where you have separate groups.  
town plots, bonus plots, road plots
1.) Road plots (Purchased plots that their only purpose to create roads) 
     (Road plots are town property and only those with roadworker perms can build or destroy on)

2.) Road plots charge a fixed tax added to the town (Base_road_tax:)

3.) Still checks if you are to close to another town. 
     (needs to be alleyed and have that both mayors permission to complete road into town)
     Haaruun-I's idea of (/t allow claimnear [PlayerName]) should be implemented regardless
     this would allow certain players to claim near your town.   

4.) Upon town, falling into ruin.  Reverts all road plots back to the original landscape. (if possible something that slowly reverts the chunk back to world seed. removing all the ores and spawners in that chunk. To keep people from abusing) 

5.) could add another rank to town RoadWorker allows players to only build and destroy in road plots
